src/tests/functional/inference_engine/snippets/src/pass/lowering.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <common_test_utils/ngraph_test_utils.hpp>
+#include "pass/lowering.hpp"
+#include "snippets/pass/collapse_subgraph.hpp"
+
+
+namespace ngraph {
+namespace builder {
+namespace subgraph {
+
+DummyTargetMachine::DummyTargetMachine() {
+    auto dummy_functor = [this](const std::shared_ptr<ngraph::Node>& n) {
+        return std::make_shared<DummyEmitter>();
+    };
+    jitters[op::v0::Parameter::get_type_info_static()] = dummy_functor;
+    jitters[op::v0::Constant::get_type_info_static()] = dummy_functor;
+    jitters[op::v0::Result::get_type_info_static()] = dummy_functor;
+    jitters[op::v1::Add::get_type_info_static()] = dummy_functor;
+    jitters[op::v1::Subtract::get_type_info_static()] = dummy_functor;
+    jitters[op::v1::Multiply::get_type_info_static()] = dummy_functor;
+    jitters[op::v1::Multiply::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::Load::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::VectorLoad::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::ScalarLoad::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::BroadcastLoad::get_type_info_static()] = dummy_functor;
+
+    jitters[ngraph::snippets::op::Store::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::VectorStore::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::ScalarStore::get_type_info_static()] = dummy_functor;
+
+    jitters[ngraph::snippets::op::Scalar::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::BroadcastMove::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::Kernel::get_type_info_static()] = dummy_functor;
+    jitters[ngraph::snippets::op::Tile::get_type_info_static()] = dummy_functor;
+}
+
+void SnippetsLoweringTests::prepare() {
```

Comment:
Not clear what `prepare` method do without implementation exploration. Can we remove `prepare()` and use `tokenize(function)` directly? `Tokenization` is term from snippets and `tokenize` method is clear.